layouts/shortcodes/dbm-postgres-agent-config-examples.md

```diff
@@ -0,0 +1,102 @@
+### One agent connecting to multiple hosts
+It is common to configure a single Agent host to connect to multiple remote database instances (see [Agent installation architectures](/database_monitoring/architecture/) for DBM). To connect to multiple hosts, create an entry for each host in the Postgres integration config.
+In these cases, it is recommended to limit to the number of instances per Agent to a maximum of 10 database instances.
+```yaml
+init_config:
+instances:
+  - dbm: true
+    host: products-primary.123456789012.us-east-1.rds.amazonaws.com
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    tags:
+      - 'env:prod'
+      - 'team:team-discovery'
+      - 'service:product-recommendation'
+  - dbm: true
+    host: products–replica-1.us-east-1.rds.amazonaws.com
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    tags:
+      - 'env:prod'
+      - 'team:team-discovery'
+      - 'service:product-recommendation'
+  - dbm: true
+    host: products–replica-2.us-east-1.rds.amazonaws.com
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    tags:
+      - 'env:prod'
+      - 'team:team-discovery'
+      - 'service:product-recommendation'
+    [...]
+```
+
+### Running custom queries
+To collect custom metrics, use the `custom_queries` option. See the sample [postgres.d/conf.yaml](https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/data/conf.yaml.example) for more details.
+```yaml
+init_config:
+instances:
+  - dbm: true
+    host: localhost
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    custom_queries:
+    - metric_prefix: employee
+      query: SELECT age, salary, hours_worked, name FROM hr.employees;
+      columns:
+        - name: age
+          type: gauge
+        - name: salary
+           type: gauge
+        - name: hours
+           type: count
+        - name: name
+           type: tag
+      tags:
+        - 'table:employees'
+```
+
+### Monitoring multiple databases 
+To specify the name of the database to connect to, use `dbname`. If omitted, the default system `postgres` database is queried. Additionally, configure `dbstrict` to `true` if you only want to gather metrics from the database provided in the `dbname` option.
+
+```yaml
+init_config:
+instances:
+  - dbm: true
+    host: localhost
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    dbname: foo  # Default: `postgres`
+    # dbstrict: true 
+  - dbm: true
+    host: localhost
```

Comment:
If they are monitoring mult dbs on the same host we don't want them to put `dbm:true` twice, since dbm pulls metrics from a global view 

I wrote some stuff about this in our internal support FAQ, that you can use for this https://datadoghq.atlassian.net/wiki/spaces/DM/pages/2645229741/PostgreSQL+FAQ#How-to-monitor-multiple-databases-with-dbm-&-the-default-pg-integration

Reply:
ah good to know, thanks!